src/app/authentication/components/form-forgot-password/form-forgot-password.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { AuthenticationService } from 'app/authentication/services/authentication.service';
+
+@Component({
+  selector: 'app-form-forgot-password',
+  templateUrl: './form-forgot-password.component.html',
+  styleUrls: ['./form-forgot-password.component.scss']
+})
+export class FormForgotPasswordComponent implements OnInit {
+  forgotPasswordForm: FormGroup;
+  alertMessage = '';
+
+  constructor(private authenticationService: AuthenticationService) {
```

Comment:
Could we add `_` for the private variable to keep align with our compass convention?

```suggestion
  constructor(private _authenticationService: AuthenticationService) {
```

![Screen Shot 2021-01-11 at 4 50 30 PM](https://user-images.githubusercontent.com/6076762/104166444-66ce7500-542d-11eb-91bd-ec289e23ecae.png)